packages/d3fc-webgl/src/series/glOhlc.js

```diff
@@ -65,11 +69,12 @@ export default () => {
         program
             .buffers()
             .attribute('aXValue', xValueAttribute)
-            .attribute('aYValue', yValueAttribute)
-            .attribute('aXDirection', xDirectionAttribute)
-            .attribute('aYDirection', yDirectionAttribute)
+            .attribute('aHigh', highAttribute)
```

Comment:
I'm not keen on the names but I can see these are copied from candlestick so that horse has bolted. I've raised #1396 to cover improving them.

Reply:
What would you have prefered them to be?